packages/spright-components/.prettierignore

```diff
@@ -1,29 +1,20 @@
 # Visit and ignore all objects in the root. 
-# Then explicitly allow all TypeScript, JavaScript, Markdown, and JSON files in the root, src, and .storybook folders.
+# Then explicitly allow all TypeScript, JavaScript, Markdown, and JSON files in the root, and src folders.
 /*
 !*.ts
 !*.js
 !*.md
-!*.mdx
 !*.json
 !src/
-!/.storybook/
 
 /src/**/*.*
 !src/**/*.ts
 !src/**/*.js
 !src/**/*.md
-!src/**/*.mdx
 !src/**/*.json
 
-/.storybook/*.*
-!/.storybook/*.js
-
 # Explicitly exclude the below files and folders
 dist
 node_modules
 package.json
 CHANGELOG.*
-src/utilities/tests/tests/fixture.spec.ts
-src/utilities/tests/fixture.ts
```

Comment:
Think we missed it in last PR but we should have a test setup in spright-components using the fixture tools. It's jasmine test specific not stroybook related